optimizer/plan/predicate_push_down.go

```diff
@@ -75,7 +49,7 @@ func (cl *columnSubstitute) Enter(inNode ast.Node) (node ast.Node, ok bool) {
 	return inNode, true
```

Comment:
`ok` -> `skipChildren`

`return inNote, false`

And the work is  done in `Leave` method, we don't need to do anything in `Enter`.